src/net/Lib/Messaging/Providers/ServiceBusQueueMessagingProvider.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Messaging.ServiceBus;
+using Lib.Data;
+
+namespace Lib.Messaging.Providers
+{
+    public class ServiceBusQueueMessagingProvider : IMessagingProvider
+    {
+        public IQueue ImageQueueClient { get; set; }
+        public IQueue ClientSyncQueueClient { get; set; }
+
+        public ServiceBusQueueMessagingProvider()
+        {
+        }
+
+        public async Task InitializeAsync(TokenCredential credential, IDataProvider dataProvider)
+        {
+            await Task.Run(() =>

```

Comment:
Other than annotating the stack trace with this method's name, no reason to `await` here.  You may want to consider returning the task directly.

Reply:
I tried to remove await and can't b/c method is async.  Did you mean new up a Task and return that and not call Run?

Reply:
You don't need `async` if you're not awaiting.   What I had in mind was something like:

```csharp
public async Task InitializeAsync(TokenCredential credential, IDataProvider dataProvider) =>
    Task.Run(() =>
    {
        var serviceBusQueueClient = new ServiceBusClient(Config.ServiceBusNamespace, credential);
        ImageQueueClient = new ServiceBusQueue(serviceBusQueueClient, Config.StorageQueueName, dataProvider);
        ClientSyncQueueClient = new ServiceBusQueue(serviceBusQueueClient, Config.StorageClientSyncQueueName, dataProvider);
    });
```